python/ray/data/_internal/arrow_block.py

```diff
@@ -266,6 +266,14 @@ def take(
         """
         return transform_pyarrow.take_table(self._table, indices)
 
+    def select(self, keys: List[KeyFn]) -> "pyarrow.Table":
+        if not all(isinstance(key, str) for key in keys):
+            raise ValueError(
+                "keys must be a list of strings when aggregating on Arrow blocks, "
+                f"but got: {type(keys)}."
```

Comment:
This will just return `but got: list`.
```suggestion
                f"but got: {[type(key) for key in keys]}."
```
Could also just dump out the `{keys}` since the repr shouldn't be too large.

Reply:
@clarkzinzow - thanks for the catch, updated to `{keys}`.